sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.NUnit/README.md

```diff
@@ -0,0 +1,125 @@
+
+Ready to get started? Jump into our [quickstart guide](#get-started)!
+
+
+## Useful Links
+- [Quickstart: Run end-to-end tests at scale](https://aka.ms/mpt/quickstart)
+- [Quickstart: Set up continuous end-to-end testing across different browsers and operating systems](https://aka.ms/mpt/ci)
+- [Explore features and benefits](https://aka.ms/mpt/about)
+- [View Microsoft Playwright Testing service demo](https://youtu.be/GenC1jAeTZE)
+- [Documentation](https://aka.ms/mpt/docs) 
+- [Pricing](https://aka.ms/mpt/pricing)
+- [Share feedback](https://aka.ms/mpt/feedback)
+
+## Get Started
+Follow these steps to run your existing Playwright test suite with the service.
+
+### Prerequisites
+
+- An Azure account with an active subscription. If you don't have an Azure subscription, [create a free account](https://aka.ms/mpt/create-azure-subscription) before you begin.
+- Your Azure account must be assigned the [Owner](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#owner), [Contributor](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#contributor), or one of the [classic administrator roles](https://learn.microsoft.com/en-us/azure/role-based-access-control/rbac-and-directory-admin-roles#classic-subscription-administrator-roles).
+- [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) must be installed in the machine from where you are running Playwright tests. 
+
+### Create a Workspace
+
+1. Sign in to the [Playwright portal](https://aka.ms/mpt/portal) with your Azure account.
+
+2. Create the Workspace.
+
+    ![Create new workspace](https://github.com/microsoft/playwright-testing-service/assets/12104064/d571e86b-9d43-48ac-a2b7-63afb9bb86a8)
+
+    |Field  |Description  |
+    |---------|---------|
+    |**Workspace Name** | A unique name to identify your workspace.<BR>The name can't contain special characters or whitespace. |
+    |**Azure Subscription** | Select an Azure subscription where you want to create the workspace. |
+    |**Region** | This is where test run data will be stored for your workspace. |
+
+  > [!NOTE]
+  > If you don't see this screen, select an existing workspace and go to the next section.
+
+### Install Microsoft Playwright Testing package
+
+1. Run this command to install the service package
+
+    ```dotnetcli
+    dotnet add package Azure.Developer.MicrosoftPlaywrightTesting.NUnit
+    ```
+
+### Setup Microsoft Playwright Testing
+
+1. Create a file `PlaywrightServiceSetup.cs` in the root directory with the below content
+
+    ```c#
+    using Azure.Developer.MicrosoftPlaywrightTesting.NUnit;
+
+    namespace PlaywrightTests; // Remember to change this as per your project namespace
+
+    [SetUpFixture]
+    public class PlaywrightServiceSetup : PlaywrightServiceNUnit;
+    ```
+
+> [!NOTE]
+> Make sure your project uses Microsoft.Playwright.NUnit version 1.37 or above.
```

Comment:
Code-fence the package name to avoid mangling during the machine translation process used for content localization.
```suggestion
> Make sure your project uses `Microsoft.Playwright.NUnit` version 1.37 or above.
```